src/pages/index.js

```diff
@@ -8,80 +8,172 @@ import { initialCards, cardData, options , cardSelector } from "../utils/constan
 import '../pages/index.css';
 import Popup from "../components/Popup";
 import { data } from "autoprefixer";
-
+import Api from "../components/Api.js"
+import PopupWithConfirmation from "../components/PopupWithConfirmation";
 
 const profileEditButton = document.querySelector("#profile-edit-button");
 const profileEditModal = document.querySelector("#profile-edit-modal");
 const profileTitle = document.querySelector("#profile-title");
 const profileDescription = document.querySelector("#profile-description");
 const profileEditForm = profileEditModal.querySelector(".modal__form");
 const profileTitleInput = document.querySelector("#edit-modal-input-title");
-const profileDescriptionInput = document.querySelector(
-  "#edit-modal-input-description"
-);
+const profileDescriptionInput = document.querySelector("#edit-modal-input-description");
 const addCardButton = document.querySelector("#profile-add-button")
 const addForm = document.querySelector("#add-form");
```

Comment:
**Could be improved**

We can get any form using their **unique** attribute `name` from `document.forms`

```js

const profileForm = document.forms["profile-form"];
const cardForm = document.forms["card-form"];

```

More info here https://developer.mozilla.org/en-US/docs/Web/API/Document/forms